src/framework/ui/dropdown/dropdown.component.tsx

```diff
@@ -226,7 +229,7 @@ class DropdownComponent extends React.Component<DropdownProps, State> {
   };
 
   private renderMenuElement = (style: StyleType): DropdownMenuElement => {
-    const { appearance, selectedOption, ...restProps } = this.props;
+    const { appearance, selectedOption, testID, ...restProps } = this.props;
```

Comment:
I don't think this is a correct way to test components.
You can use any of [getAllBy](https://callstack.github.io/react-native-testing-library/docs/api-queries#getallby) queries to find a corresponding MenuElement

Reply:
Refactored